editor/styling/stylesheets/components/_scroll-bar.scss

```diff
@@ -1,5 +1,5 @@
 .scroll-bar {
-    -fx-background-color: -df-background;
+    -fx-background-color: -df-background-light;
```

Comment:
I think someone attempted to "clean up" the CSS a bit at some point and broke it. The scroll bars should now look good again.